# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ggrab/00-makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur ggrab-0.21/Makefile patched-ggrab-0.21/Makefile
--- ggrab-0.21/Makefile	2003-01-06 13:09:20.000000000 +0100
+++ patched-ggrab-0.21/Makefile	2003-02-21 14:47:59.000000000 +0100
@@ -13,12 +13,12 @@
 #
 #
 
-CPPFLAGS=-DREENTRANT -D_LARGEFILE64_SOURCE  -Wall -Wshadow -O3
+CPPFLAGS=-DREENTRANT -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -Wshadow -O3
 
-LDFLAGS= -s -static
+LDFLAGS= -s
 
 INSTALL=/usr/bin/install 
-DESTDIR=/usr/local
+DESTDIR=/usr
 
 TARGETS=ggrab sserver
 
